test/es6/bug_OS14880030.js

```diff
@@ -0,0 +1,21 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
```

Comment:
Can you add testing that the cases that should throw still do?

Reply:
Sure. I added a couple which should exercise those cases.